Detectors/TOF/simulation/src/Digitizer.cxx

```diff
@@ -783,10 +799,12 @@ void Digitizer::fillOutputContainer(std::vector<Digit>& digits)
   // filling the digit container doing a loop on all strips
   for (auto& strip : *mStripsCurrent) {
     strip.fillOutputContainer(digits);
+    if (strip.getNumberOfDigits())
+      printf("strip size = %d - digit size = %d\n", strip.getNumberOfDigits(), digits.size());
```

Comment:
look like a debug message, can this be removed or converted to ``LOG(DEBUG)``?

Reply:
DONE (visible soon in the PR)